lib/dealership.rb

```diff
@@ -0,0 +1,37 @@
+class Dealership
+  attr_reader :name, :address, :inventory
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+  end
+
+  def inventory_count
+    @inventory.length
+  end
+
+  def add_car(car)
+    @inventory << car
+  end
+
+  def has_inventory?
+    if inventory_count > 0
+      true
+    else
+      false
+    end
+  end
+
+  def cars_by_make(make)
+    same_make = []
+    @inventory.each do |inventory|
+      if inventory.car.make == make
+      @same_make << @inventory.car
```

Comment:
Careful of your `@` character here - `same_make` is not a class-level variable, it is a method variable that you created on line 26, and so does not need the @. Same with `@inventory` - this is inside of your .each, so you want to add the actual `inventory` block variable (which again is technically a Car since we're looping through) to this array.